src/Data/ProgrammesDb/Type/UtcDateTimeType.php

```diff
@@ -38,17 +40,17 @@ public function convertToPHPValue($value, AbstractPlatform $platform): ?Chronos
         if (is_null($value) || $value instanceof Chronos) {
             return $value;
         }
-
-        $converted = Chronos::createFromFormat($platform->getDateTimeFormatString(), $value, 'UTC');
-
-        if (!$converted) {
-            throw ConversionException::conversionFailedFormat(
-                $value,
-                $this->getName(),
-                $platform->getDateTimeFormatString()
-            );
+        $dateFormatString = $platform->getDateTimeFormatString();
+        if (!isset(self::$cache[$dateFormatString][$value])) {
```

Comment:
We've already checked for null. It's going to be null or a string. This blows up with an exception if its not anyway.